gaepypi/storage.py

```diff
@@ -20,13 +20,16 @@
 
 import six
 from abc import ABCMeta, abstractmethod
-import cloudstorage as gcs
+from google.cloud import storage
+from io import BytesIO
 
-my_default_retry_params = gcs.RetryParams(initial_delay=0.2,
-                                          max_delay=5.0,
-                                          backoff_factor=2,
-                                          max_retry_period=15)
-gcs.set_default_retry_params(my_default_retry_params)
+storage_client = storage.Client()
+
+
+def to_bucket_and_path(p):
+	segments = p.lstrip('/').split('/')
+	bucket_path = '/'.join(segments[1:])
+	return segments[0], bucket_path
```

Comment:
`segments[0]` never seem to be used it'a always `_, bucket_path = to_bucket_and_path(path)`, so why do we return it?